tex2pdf_service/Makefile

```diff
@@ -2,7 +2,7 @@ app_tag := arxiv-tex2pdf-app
 app_name := tex2pdf_app
 app_port := 6301
 dockerport := 8080
-APP_DOCKER_RUN := docker run --cpus 2 --rm -p ${app_port}:${dockerport} -e PORT=${dockerport} --name ${app_name} --security-opt="no-new-privileges=true" 
+APP_DOCKER_RUN := docker run --cpus 2 --rm -p ${app_port}:${dockerport} -e PORT=${dockerport} -e WORKERS=16 --name ${app_name} --security-opt="no-new-privileges=true" 
```

Comment:
Hard-coding the worker numbers is not a good idea. If this should run on some cloud instance, we don't have access to that many workers I fear.

Reply:
This is for the local run only. The cloud service has entirely different set up in CICD.